orchestrators/InsightOperations/tests/Unit/Coordinators/GovernanceNarrativeCoordinatorTest.php

```
@@ -0,0 +1,102 @@
+            $cachePort,
+            $availabilityPort,
+            $narrativePort,
+        );
+
+        $result = $coordinator->show("tenant-a", "vendor-1")->toResponseArray();
+
+        self::assertSame(0, $narrativePort->calls);
+        self::assertSame("vendor-1", $result["data"]["vendor_id"]);
+        self::assertCount(1, $result["data"]["evidence"]);
+        self::assertFalse($result["data"]["ai_narrative"]["available"]);
+    }
```

PHPMD (2.15.0)

[error] The method test_show_returns_governance_facts_and_cached_narrative is not named in camelCase. (undefined)

(CamelCaseMethodName)